Websocket/socket.ts

```
@@ -638,14 +640,14 @@ export function setupWebSocketServer(wss: WebSocketServer, instanceId?: string)
           },
           emit: (event: string, ...args: unknown[]) => {
             if (listeners[event])
-              listeners[event].forEach((cb) => cb(...(args as Parameters<EventCallback>)));
+              listeners[event].forEach((cb) => cb(...args));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)